core/Updates/4.3.0-b3.php

```diff
@@ -36,13 +36,6 @@ public function getMigrations(Updater $updater)
         $migrations = [];
 
         $migrations[] = $this->migration->db->addColumn('segment', 'hash', 'CHAR(32) NULL', 'definition');
-
-        $segmentTable = Common::prefixTable('segment');
-        $segments = Db::fetchAll("SELECT idsegment, definition FROM $segmentTable WHERE deleted = ?", [0]);
-        foreach ($segments as $segment) {
-            $hash = md5(urldecode($segment['definition']));
-            $migrations[] = $this->migration->db->sql("UPDATE `$segmentTable` SET `hash` = '$hash' WHERE `idsegment` = '{$segment['idsegment']}'");
-        }
```

Comment:
Removing this one, as otherwise all segments might be updated twice when updating from a version prior 4.3.0-b3